aspnet-core/src/toyiyo.todo.Web.Mvc/wwwroot/js/signalr-manager.js

```
@@ -0,0 +1,21 @@
+let app = app || {};
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)